modules/atf/util.lua

```diff
@@ -145,7 +145,8 @@ function module.heartbeat(str)
 end
 
 function module.sdl_core(str)
-  if (not is_file_exists(str.."smartDeviceLinkCore")) then
+  if (not is_file_exists(str.."smartDeviceLinkCore")) and 
+     (not is_file_exists(str.."/smartDeviceLinkCore")) then
```

Comment:
@LuxoftAKutsan It is moved from previous commit after rebase. It is already in develop